recipes/prophyle/meta.yaml

```diff
@@ -10,7 +10,7 @@ package:
   #sha256:
 
 build:
-  number: 0
+  number: 1
   skip: True # [py27]
   script: pip install --upgrade git+https://github.com/karel-brinda/prophyle
```

Comment:
Can we please change this recipe to use the `source` section and not github. Thanks.